src/django_components/slots.py

```diff
@@ -368,99 +462,150 @@ def _resolve_slot_context(self, context: Context, slot_fill: "SlotFill") -> Cont
             raise ValueError(f"Unknown value for context_behavior: '{registry.settings.context_behavior}'")
 
-    def resolve_kwargs(
-        self,
-        context: Context,
-        component_name: Optional[str] = None,
-    ) -> Tuple[str, Dict[str, Optional[str]]]:
-        _, kwargs = self.params.resolve(context)
-        name = kwargs.pop(SLOT_NAME_KWARG, None)
 
-        if not name:
-            raise RuntimeError(f"Slot tag kwarg 'name' is missing in component {component_name}")
+class FillNode(BaseNode):
+    """
+    Use this tag to insert content into component's slots.
 
-        return (name, kwargs)
+    `{% fill %}` tag may be used only within a `{% component %}..{% endcomponent %}` block.
+    Runtime checks should prohibit other usages.
 
+    **Args:**
 
-class FillNode(BaseNode):
-    """Node corresponding to `{% fill %}`"""
+    - `name` (str, required): Name of the slot to insert this content into. Use `"default"` for
+        the default slot.
+    - `default` (str, optional): This argument allows you to access the original content of the slot
+        under the specified variable name. See
+        [Accessing original content of slots](../../concepts/fundamentals/slots#accessing-original-content-of-slots)
+    - `data` (str, optional): This argument allows you to access the data passed to the slot
+        under the specified variable name. See [Scoped slots](../../concepts/fundamentals/slots#scoped-slots)
 
-    def __init__(
-        self,
-        nodelist: NodeList,
-        params: TagParams,
-        trace_id: str,
-        node_id: Optional[str] = None,
-    ):
-        super().__init__(nodelist=nodelist, params=params, node_id=node_id)
+    **Examples:**
 
-        self.trace_id = trace_id
+    Basic usage:
+    ```django
+    {% component "my_table" %}
+      {% fill "pagination" %}
+        < 1 | 2 | 3 >
+      {% endfill %}
+    {% endcomponent %}
+    ```
 
-    def render(self, context: Context) -> str:
-        if _is_extracting_fill(context):
-            self._extract_fill(context)
-            return ""
+    ### Accessing slot's default content with the `default` kwarg
 
-        raise TemplateSyntaxError(
-            "FillNode.render() (AKA {% fill ... %} block) cannot be rendered outside of a Component context. "
-            "Make sure that the {% fill %} tags are nested within {% component %} tags."
-        )
+    ```django
+    {# my_table.html #}
+    <table>
+      ...
+      {% slot "pagination" %}
+        < 1 | 2 | 3 >
+      {% endslot %}
+    </table>
+    ```
 
-    def __repr__(self) -> str:
-        return f"<{self.__class__.__name__} ID: {self.node_id}. Contents: {repr(self.nodelist)}.>"
+    ```django
+    {% component "my_table" %}
+      {% fill "pagination" default="default_pag" %}
+        <div class="my-class">
+          {{ default_pag }}
+        </div>
+      {% endfill %}
+    {% endcomponent %}
+    ```
 
-    def resolve_kwargs(self, context: Context) -> "FillWithData":
```

Comment:
`FillNode.resolve_kwargs` and `FIllNode._process_kwarg` were removed. Now There is only `FillNode.render()`